src/components/content/advanced-asset/advanced-asset.tsx

```diff
@@ -1,49 +1,40 @@
 import { GatsbyImage } from 'gatsby-plugin-image';
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import Zoom from 'react-medium-image-zoom';
 import styled from 'styled-components';
+import customComponents from '../../legacy/markdown/custom-components';
 
-interface ContainerProps {
-  $width: string;
-}
-
-const FullWidthContainer = styled.div.attrs((props: ContainerProps) => props)`
-  width: ${(props) => props.$width};
-  position: relative;
-  left: 50%;
-  right: 50%;
-  margin-left: -50vw;
-  margin-right: -50vw;
+const FullWidthContainer = styled.div`
+  width: 100%;
+  && {
```

Comment:
- Why is the `&&` needed?
- Please add an empty line between those 2 lines, if the selector is still needed. CSS needs some space to breath ;-)

Reply:
Ohps, it is not needed => 🧹